pmm-app/package.json

```diff
@@ -18,6 +18,7 @@
     "dev": "grafana-toolkit plugin:dev",
     "watch": "grafana-toolkit plugin:dev --watch",
     "e2e": "docker run --net=host -v $PWD:/tests -v $PWD/node_modules:/node_modules --env-file env.generated.list codeception/codeceptjs:2.6.1 codeceptjs run-multiple parallel --all --debug --steps -c pr.codecept.js --grep '(?=.*)^(?!.*@visual-test)^(?!.*@not-pr-pipeline)'",
+    "e2e:local": "docker run --net=host -v $PWD:/tests -v $PWD/node_modules:/node_modules codeception/codeceptjs:2.6.1 codeceptjs run-multiple parallel --all --debug --steps -c pr-local.codecept.js --grep '(?=.*)^(?!.*@visual-test)^(?!.*@not-pr-pipeline)'",
```

Comment:
we removed this tag visual-test, it was replaced with `@not-ui-pipeline` could you please replace on `e2e` and `e2e:local` both